### PR TITLE
Add existing project

### PR DESCRIPTION
**Pull request by emirsagit**
**Add existing project**

- Setup lint and change README.md
- Copied my code from an existing project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+          <button
+            class="nav-link btn btn-secondary"
+            onclick="goto('video')"
+            type="button"
+          >
+            ABOUT
+          </button>
+        </li>
+      </ul>
+    </div> -->
```
Comment (emirsagit) [inline index.html:98]: Please remove commented lines for a cleaner codebase.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    </div>
+
+    <div
+      id="video"
+      class="p-4 d-flex justify-items-center align-items-center flex-column margin-top-100 mb-5"
+    >
+      <h3 class="text-center py-3">ABOUT US</h3>
+    </div>
+
+    <script>
```
Comment (emirsagit) [inline index.html:195]: Please add your javascript to a separate file. Please name the javascript file as app.js.

**Review by emirsagit — COMMENTED** [review]
Change required :recycle: 

It's a good job :+1: 

But there are a few little changes required.
Please check my comments on index.html file. Make all changes.
Then submit for re-review.

Thanks :clap:

**Review by bolah2009 — APPROVED** [review]
Hi @emirsagit  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (bolah2009) [reply index.html:195]: I would recommend you add reasons "why" a separate file is better and why it should be named `app.js`. 🙂

**Pull request merged**